_posts/2019-01-23-lecture-03.md

```diff
@@ -0,0 +1,470 @@
+
+$$
+P(X_1,\cdots,X_n) = \frac{1}{Z}f(X,Z)g(Y,Z),
+$$
+
+and we observe that independence follows.
+
+If $$X\cup Y\cup Z$$ is a strict subset of $$\mathcal{X}$$. Let $$U = \mathcal{X}\setminus (X\cup Y\cup Z)$$. We can partition $$U$$ into two disjoint sets $$U_1$$ and $$U_2$$ such that $$Z$$ separates $$X\cup U_1$$ from $$Y\cup U_2$$ in $$\mathcal{G}$$. Using our argument from the partition case, we have that $$\big((X\cup U_1)\perp ((Y\cup U_2)\vert Z$$. Apply decomposition property of probability we attain that $$P\models (X\perp Y\vert Z)$$.
+
+### Completeness (Hammesley-Clifford theorem)
+
+Let $$P$$ be a positive distribution over $$\mathcal{X}$$, and $$\mathcal{G}$$ a Markov network graph over $$\mathcal{X}$$. If $$\mathcal{G}$$ is an I-map for $$P$$, then $$P$$ is a Gibbs distribution that factorizes over $$\mathcal{G}$$.
+
+This result shows that, for positive distributions, the global independencies imply that the distribution factorizes according to the network structure. Thus, for this class of distributions, we have that a distribution $P$ factorizes over a Markov network $$\mathcal{G}$$ if and only if $$\mathcal{G}$$ is an I-map for $$P$$.
+
+## Other Markov Properties
+
+For UGMs, we defined I-maps in terms of global Markov properties. We will now define local independence. Intuitively, when two variables are not directly linked, there must be some way of rendering them conditionally independent. Specifically, we can require that $X$ and $Y$ be independent given all other nodes in the graph.
+
+### Definition: Pairwise Independencies
+
+Let $$\mathcal{G}$$ be a Markov network. We define the pairwise independencies associated with $$\mathcal{G}$$ to be
+
+$$
+\mathcal{I}_P(\mathcal{G}) = \{(X\perp Y\vert \mathcal{X}-\{X,Y\}): X-Y\notin \mathcal{G}\}
+$$
+
+<figure id="pairwise-indep" class="l-body">
+  <div class="row">
+    <div class="col two">
+      <img src="{{ 'assets/img/notes/lecture-03/graph_local_example.png' | relative_url }}" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Example of pairwise independence</strong>
+  </figcaption>
+</figure>
+
+The figure above demonstrates that $$A\perp B\vert \{B,C,D\}$$. 
+
+Using the definition of pairwise independence and local independence, we attain that
+
+### Proposition
+
+For any Markov network $$\mathcal{G}$$ and any distribution $$P$$, we have that if $$P\models \mathcal{I}_l(\mathcal{G})$$  then $$P\models \mathcal{I}_P(\mathcal{G})$$.
+
+### Proposition
+
+For any Markov network $$\mathcal{G}$$ and any distribution $$P$$, we have that if $$P\models \mathcal{I}(\mathcal{G})$$ then $$P\models \mathcal{I}_l(\mathcal{G})$$.
+
+Finally, we reach at the following conclusion
+
+### Proposition
+
+Let $$P$$ be a positive distribution. If $$P$$ satisfies $$\mathcal{I}_P(\mathcal{G})$$, then $$P$$ satisfies $$\mathcal{I}(\mathcal{G})$$.
+
+### Theorem
+
+The followings are equivalent:
```

Comment:
This is only true for positive distributions. Please add "The following statements are equivalent for a positive distribution $$P$$:"